tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/RabbitMQ/ContextPropagation.cs

```diff
@@ -12,12 +12,12 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.RabbitMQ
 {
     internal static class ContextPropagation
     {
-        public static void HeadersSetter(IDictionary<string, object> carrier, string key, string value)
+        public static readonly Action<IDictionary<string, object>, string, string> HeadersSetter = (carrier, key, value) =>
```

Comment:
I'm sure it's _not_ necessary here, but it makes me feel happy

```suggestion
        public static readonly Action<IDictionary<string, object>, string, string> HeadersSetter = static (carrier, key, value) =>
```

Reply:
Added in 31aee89. I cannot see any changes in the codegen as expected.